docs/source/backends/openvpn.rst

```diff
@@ -171,6 +171,8 @@ Required properties:
 +--------------------------+---------+--------------+-------------------------------------------------------------+
 | ``verb``                 | integer | ``1``        | from ``0`` (disabled) to ``11`` (very verbose)              |
 +--------------------------+---------+--------------+-------------------------------------------------------------+
+| ``tls_auth``             | string  |              | string containing TLS Auth key                              |
```

Comment:
let's move this after `tls_cipher`, in general, let's add new options where they make sense, not just at the end.